tests/tools/create_chunked_dataset.py

```diff
@@ -15,7 +15,7 @@
 
 
 def create_chunked_dataset(
-    root_dir, num_chunks, include_masks=False, include_edge_data=False
+    root_dir, num_chunks, include_masks=False
```

Comment:
Maybe keep this option so you can test two cases: w/ and w/o edge data

Reply:
this is a temporary argument that was added on purpose to bypass possible test failure caused by `decouple of num_chunks and num_parts` not supported for edge feature yet. It's ok to remove it.

Reply:
What I mean is to test two cases (w/ and w/o edge data) separately to ensure that there is no bug when the data does not have any edge data. You can ignore me if you've already tested it.

Reply:
I have tested it...